bin/entrypoint

```diff
@@ -17,6 +17,6 @@ elif [ "$1" = "run" ]; then
     jetstream --log_to_bigquery run "${@:2}"
 else
     # this is triggered by Airflow
-    jetstream --log_to_bigquery run "$@" --argo
```

Comment:
Actually, I don't think we need to handle this since it shouldn't get invoked from anywhere. 
The `run` command has no `--argo` flag, instead `run-argo` should be used. So this line is wrong